_posts/2024-11-11-test-markdown.md

```
@@ -0,0 +1,157 @@
+
+func NewLocalKeyValue(key string, value string, revision int64) *LocalKeyValue {
+	return &LocalKeyValue{
+		Key:      key,
+		Value:    value,
+		Revision: revision,
+	}
+}
+
+
+func watchKey(cli *clientv3.Client, lkv *LocalKeyValue) {
+	rch := cli.Watch(context.Background(), lkv.Key, clientv3.WithRev(lkv.Revision+1))
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)